### PR TITLE
ignore_params now works with menu="no"

### PR DESCRIPTION
**Pull request by RossLote**
If you set ignore params="yes" menu="no" and the full_path contains params then the paths won't match. Here is a fix for it.

### CODE REVIEW

**Comment by hellysmile** [comment]
Hey @tony , are You interested to take a look on it?

**Comment by hellysmile** [comment]
I have a question. Is it technically possible that fragment will be in full_path?

Btw, we might need to ignore fragment in href as well

**Comment by RossLote** [comment]
I don't think it is possible for full_path to have fragments but I thought I'd put it in there just in case.
The fragment is already removed from href [here](https://github.com/hellysmile/django-activeurl/blob/dfc561cc265b3a4a7377cfd33ba85015d86282c0/django_activeurl/utils.py#L116)

**Comment by hellysmile** [comment]
Can we do a small test regarding full_path and fragment? If it is impossible then lets remove fragment replacing

**Comment by tony** [comment]
I'm fine with this PR.

Clarification: Is the issue w/ fragment related to `menu` separate? It seems unrelated.

On that note, isn't it possible `url.attrib.get('href', None)` could pull in a fragment after the URL? The user may be linking to something with an internal anchor.

**Comment by tony** [comment]
OK ok now I see what you mean @hellysmile. Let me comment in the code.

File: django_activeurl/utils.py [hunk 1]
```diff
@@ -118,6 +118,10 @@ def check_active(url, element, **kwargs):
         # cut off get params (?key=var&etc=var2)
         if ignore_params:
             href = href._replace(query='')
+            full_path = urlparse.urlsplit(
+                kwargs['full_path']
+            )._replace(query='', fragment='')
```
Comment (tony) [inline django_activeurl/utils.py]: fragment='' can be removed

**Comment by tony** [comment]
Hold up a second, I'm confused. 

Why is `kwargs['full_path']` being used?

Can you explain what that's doing?

**Comment by RossLote** [comment]
@tony I'm not sure what you mean? What else would be there? I'm removing the querystring from the full path.

**Comment by RossLote** [comment]
When `kwargs['full_path']='/page/?foo=bar'`, `href='/page/'` and `menu="no"` then the logic check will always fail as it checks for an exact match. So when `ignore_params="yes"` is on both `kwargs['full_path']` AND `href` should have their querystrings stripped.

**Comment by tony** [comment]
That helps @RossLote 

I'm overwhelmed by all the variables being thrown around `check_active` and its lack of documentation. So it's not you

Any way you can do it without a one-time variable being created? I see `full_path` is created, is it used elsewhere?

**Comment by RossLote** [comment]
@tony. That was just for readability, it's gone now.

**Comment by tony** [comment]
In addition to the CI tests, I also tested this locally. Doesn't break anything on my end.

I don't think I have pypi access, do I @hellysmile ?  Can you tag a minor release for this?

**Comment by hellysmile** [comment]
0.1.11 will be pushed to pypi today! Thanks guys! Great job!

**Pull request merged**